packages/db/src/core/integration/index.ts

```diff
@@ -31,14 +33,13 @@ function astroDBIntegration(): AstroIntegration {
 		hooks: {
 			'astro:config:setup': async ({ updateConfig, config, command: _command, logger }) => {
 				command = _command;
+				root = config.root;
 				if (_command === 'preview') return;
 
 				let dbPlugin: VitePlugin | undefined = undefined;
-				const studio = config.db?.studio ?? false;
 
-				if (studio && command === 'build' && process.env.ASTRO_DB_TEST_ENV !== '1') {
```

Comment:
The test suite was updated to test the dev server, so this env flag was removed. For production builds, we should instead stand up a proxy URL and override using the `ASTRO_STUDIO_REMOTE_DB_URL` variable. Testing a local sqlite file in production was not mirroring the expected user flow.